python/tests/test_highlevel.py

```diff
@@ -1272,6 +1272,28 @@ def test_at(self):
                     self.assertEqual(t3.interval, t2.interval)
                     self.assertEqual(t3.parent_dict, t2.parent_dict)
 
+    def test_sequence_iteration(self):
+        for ts in get_example_tree_sequences():
+            for table_name, _ in ts.tables:
+                sequence = getattr(ts, table_name)()
+                length = getattr(ts, "num_" + table_name)
+                # Test __iter__
+                for i, n in enumerate(sequence):
+                    self.assertEqual(i, n.id)
+                self.assertEqual(n.id, length - 1 if length else 0)
+                if table_name != 'mutations':
```

Comment:
We should test something about the mutation table. Does len() fail, etc?

Reply:
Tested for failure of len() and indexing. Testing for access by index (square braces) is within a context manager.